infra/core/compute/ubuntu-jumpbox.bicep

```diff
@@ -189,7 +171,7 @@ resource virtualMachine 'Microsoft.Compute/virtualMachines@2021-11-01' = {
       osDisk: {
         createOption: 'FromImage'
         managedDisk: {
-          storageAccountType: osDiskType
+          storageAccountType: 'Standard_LRS'
```

Comment:
Why did we move this from a variable?

Reply:
this along with other variables that were only declared once and then referenced here. made a point of indirection that made readibility hard. if we end up needing to reuse it elsewhere, we can use the variables again